cognite_toolkit/_cdf_tk/loaders/_data_loaders.py

```diff
@@ -209,9 +217,6 @@ def upload(self, state: BuildEnvironment, ToolGlobals: CDFToolConfig, dry_run: b
                 )
                 continue
 
-            if table.table_name is None:
-                # This should never happen
-                raise ValueError(f"Missing table name for {datafile.name}")
```

Comment:
We use an identifier that has table_name: str and not str | None. So `mypy` will pick up if there are any issues anywhere.